cryptofeed/exchanges/binance.py

```diff
@@ -454,5 +454,7 @@ async def subscribe(self, conn: AsyncConnection):
         # Binance does not have a separate subscribe message, the
         # subscription information is included in the
         # connection endpoint
-        if not isinstance(conn, HTTPPoll):
+        if isinstance(conn, (HTTPPoll, HTTPConcurrentPoll)):
+            self.open_interest = {}
```

Comment:
there isnt currently a defined `open_interest` method in the RestExchange class or the RestMixins, but there probably will be soon. Naming should be changed to _open_interest. I know this was here before you modified it, but now is as good a time as any to change it

Reply:
this no longer applies, I changed the new to self._open_interest_cache, so just update the name here